src/index.js

```diff
@@ -1,12 +1,10 @@
 import express from 'express';
-import config from './config.js';
 import fileUpload from 'express-fileupload';
 
 
 export default class Server {
-    constructor (port, host) {
-        this.port = port || config.port;
-        this.host = host || config.host;
+    constructor (conf) {
+        this.config = conf;
```

Comment:
This way there is no option to create Server without config at all.
Let our server use default config if `config` arg is not set